packages/atomic/src/components/search/facets/atomic-numeric-facet/atomic-numeric-facet.tsx

```diff
@@ -197,13 +231,20 @@ export class AtomicNumericFacet implements InitializableComponent {
   }
 
   public initialize() {
+    if (this.tabsIncluded.length > 0 && this.tabsExcluded.length > 0) {
```

Comment:
Mh should the warning be when tabsIncluded & tabsExcluded are both specified?
What I had in mind was more if there's an intersection between the two sets/arrays.

Reply:
@louis-bompart I don't think there's any situation where it would make sense to both include and exclude, since only one tab can be active at a time...